src/iblt.cpp

```diff
@@ -117,9 +117,14 @@ void CIblt::_insert(int plusOrMinus, uint64_t k, const std::vector<uint8_t> v)
 {
     // assert(v.size() == valueSize);
 
+    if (!n_hash)
+        return;
+    size_t bucketsPerHash = hashTable.size() / n_hash;
+    if (!bucketsPerHash)
```

Comment:
Similar to n_hash, hashTable is only resized in resize and reset. Why don't we raise an assertion error or runtime exception when it is zero? Again, the IBLT is non-functional with an empty hashTable.